src/index.html

```diff
@@ -1,16 +1,98 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+        rel="stylesheet"
+        href="./reset.css"
+      >
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container--l">
+      <form
+        class="form"
+        method="get"
+        data-qa="big"
+        action="/"
+      >
+        <label class="big-search__button button-general" for="bigSearchInput">
+          <svg
```

Comment:
it will be clearer to use a separate file for svg.

Reply:
+